android/versioned-abis/expoview-abi33_0_0/build.gradle

```diff
@@ -41,7 +41,7 @@ android {
     versionCode 1
     versionName "1.0"
     ndk {
-      abiFilters 'armeabi-v7a', 'x86'
+      abiFilters 'armeabi-v7a', 'x86', 'arm64-v8a', 'x86_64'
```

Comment:
Does it make sense to make this change?

Reply:
I assumed that when we run update-exponent-view we are using this build gradle. Am I wrong?